cloudpickle/cloudpickle.py

```diff
@@ -865,6 +871,51 @@ def save_weakset(self, obj):
 
         dispatch[weakref.WeakSet] = save_weakset
 
+    if HAVE_ENUM:
+
+        def save_enum_instance(self, obj):
+            """Save an **instance** of enum.Enum.
+
+            This is what gets called to save enum members.
+            """
+            self.save_reduce(getattr, (type(obj), obj.name))
```

Comment:
Ah, good catch.  It's easy enough to add coverage for this case if we decide we want this feature, though I'm not totally convinced it's a good idea because idiomatic usage of `Enum` involves lots of `is` comparisons, which will break silently if a user unpickles the same Enum multiple times.